rs/nervous_system/integration_tests/BUILD.bazel

```
@@ -344,3 +348,28 @@ rust_test(
     ],
     deps = [":nervous_system_integration_tests"] + DEPENDENCIES_WITH_TEST_FEATURES + DEV_DEPENDENCIES,
 )
+
+rust_test(
+    name = "sns_extension_test",
+    timeout = "long",
+    srcs = [
+        "tests/sns_extension_test.rs",
+    ],
+    aliases = ALIASES,
+    data = DEV_DATA + [
+        "@kong_backend_canister//file",
+        "@kongswap-adaptor-canister//file",
+    ],
+    env = DEV_ENV | {
+        "KONG_BACKEND_CANISTER_WASM_PATH": "$(rootpath @kong_backend_canister//file)",
+        "KONGSWAP_ADAPTOR_CANISTER_WASM_PATH": "$(rootpath @kongswap-adaptor-canister//file)",
+        "RUST_TEST_NOCAPTURE": "1",
+    },
+    proc_macro_deps = MACRO_DEPENDENCIES + MACRO_DEV_DEPENDENCIES,
```

I see `MACRO_DEV_DEPENDENCIES` is empty so better to remove it to not accidentally introduce unused dependencies when folks add things to it that are only necessary in other targets.
```suggestion
    proc_macro_deps = MACRO_DEPENDENCIES,
```